Makefile

```diff
@@ -53,9 +53,36 @@ docker: build
 	docker build cmd/osdslet -t opensdsio/opensds-controller:latest
 	docker build cmd/osdsapiserver -t opensdsio/opensds-apiserver:latest
 
-test: build
+test: build osds_verify osds_unit_test osds_integration_test osds_e2eflowtest_build osds_e2etest_build
 	script/CI/test
```

Comment:
Are these flags duplicated with `script/CI/test`？

Reply:
Yes.. all these test tasks are segragated now to different scripts